python/lsst/ts/m1m3/utils/thermocouples.py

```diff
@@ -52,318 +56,758 @@ async def get_scanner_data(
         Astropy Time for end of query window.
     time_bin : `int`
         Time bin in seconds (thermocouple records data every 30s
-        so a bin smaller than 30s is not recommended).
-        default: 30
-    do_remove_cold_junction : `boolean`, optional
-        If true, remove the cold junction offset for the thermal scanner.
-        Defaults to True.
-    do_remove_offsets : `boolean`, optional
-        If true, remove offsets for each thermocouple.
-        Defaults to True.
-
-    Returns
-    -------
-    scanner_dataframe : `DataFrame`
+        so a bin smaller than 60s is not recommended).
+        default: 60
+
+    Attributes
+    ----------
+    efd_client : EfdClient
+        EFD client to use for the EFD queries
+    cold_junction_dataframe : pd.DataFrame
+        Time binned dataframe of EFD temperatures where the index
+        is time and the columns are cold junction temperature.
+    all_thermocouples_dataframe : pd.DataFrame
         Time binned dataframe of EFD temperatures where the index
         is time and the columns are thermocouple temperature.
+    vertical_gradient_dataframe : pd.DataFrame
+        Time binned dataframe of EFD temperatures where the index
+        is time and the columns are difference between the front and back
+        temperature of the mirror.
+    nonstandard_thermoocouples : List
+        List of thermocouples that are in cells with nonstandard air
+        nozzle configurations.
+    xy_r_gradients : pd.DataFrame
+        Time binned dataframe of EFD temperatures where the index
+        is time and the columns are x, y, and radial gradients across
+        the mirror in deg C/m.
+    vertical_xy_r_gradients : pd.DataFrame
```

Comment:
I'm not sure if this quantity is physical/useful....it uses a subset of cells (only the ones with F/B pairs) to calculate the x/y/r gradients of a quantity that is the per-cell F/B difference. I don't think that's the same as the x/y/r gradients produced by the full 3D calculation and I think those are the ones that matter.